executor/include/execution/trait/is_executor.hpp

```diff
@@ -0,0 +1,27 @@
+//
+// Created by Shrijit Singh on 14/06/20.
+//
+
+#pragma once
+
+#include <experimental/type_traits>
+#include <type_traits>
+
+namespace execution {
+
+template <class Executor, typename = void>
+struct is_executor : std::false_type {};
+
+constexpr const auto noop = [] {};
```

Comment:
This is an implementation detail. Hide it please, or refactor so that is not needed

Reply:
Will be switching to the properties mentioned [here](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1660r0.pdf). So won't be needing this anymore.

Reply:
https://github.com/shrijitsingh99/unified-executors/pull/7/commits/be448a9448971a7db8b36fa0bd5bee997cc346d4
Added to my TODO